README.md

```diff
@@ -46,16 +46,21 @@ node validate.js
 
 #### Command usage:
 ```
-D:\sdk\openapi-validation-tools>node validate.js
-
-Usage: node validate.js <command> <spec-path> [--json]
-
-
+MacBook-Pro:openapi-validation-tools someUser$ node cli.js -h
```

Comment:
this is just a starter file for now. In the future we will add a bash script in the bin folder. It will be very similar to the current xplat cli structure.

For example in xplat cli all the commands start with `azure something something`

We can come up with a nice name something like `oav validate-spec <spec-path> -j`
where oav - Open Api Validation. (just a suggestion ;).. We can put anything that the project captain likes)